lib/compiler.js

```diff
@@ -71,6 +71,10 @@ module.exports = function compile(options) {
 
   bootDirs.forEach(function(dir) {
     bootScripts = bootScripts.concat(findScripts(dir));
+    if (env) {
```

Comment:
If `env` is `null`, then the `||` operator will pick the next value.

```
$ node
> null || 'development'
'development'
> 
```

Thus if you specify `env: null`, then `development` will be used instead.